packages/skin/src/sass/video/video.scss

```diff
@@ -103,3 +103,58 @@
 .video-player .shaka-video-container {
     font-family: variables.$font-family-default;
 }
+
+.video-player--with-nav .shaka-controls-container {
+    pointer-events: none;
+    .shaka-bottom-controls,
+    .shaka-play-button-container {
+        pointer-events: auto;
+    }
+}
+
+.video-player--compact {
+    container-type: inline-size;
+    width: 100%;
+    .shaka-controls-button-panel {
+        background-color: transparent;
+        justify-content: flex-end;
+        pointer-events: none;
+        & > :first-child {
+            border-radius: 16px;
+            font-size: variables.$font-size-medium;
+            margin-right: auto;
+            padding: 16px 8px;
+            width: auto;
+        }
+        button {
+            * {
+                margin: 0;
+            }
+
+            background-color: rgb(0 0 0 / 0.32);
+            border-radius: 50%;
+            margin: 5px;
+            &:enabled {
+                pointer-events: auto;
+            }
+        }
+    }
+    .shaka-scrim-container {
+        background: none;
+    }
+    .shaka-tooltip:hover::after {
+        visibility: hidden;
+    }
+    @container (inline-size < #{variables.$screen-size-XS}) {
```

Comment:
Container queries are still _just_ outside our browser policy (looks like they were added in Safari 16). Are we okay with this being a progressive enhancement?